<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Intake</class>
 <widget class="QMainWindow" name="Intake">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>295</width>
    <height>338</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Intake</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QListWidget" name="listShipments">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>231</width>
      <height>251</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>URW Gothic L</family>
      <pointsize>16</pointsize>
     </font>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>295</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="quit"/>
   </widget>
   <widget class="QMenu" name="menuIntake">
    <property name="title">
     <string>Intake</string>
    </property>
    <addaction name="newBag"/>
    <addaction name="actionPost_Test"/>
   </widget>
   <widget class="QMenu" name="menuPages">
    <property name="title">
     <string>Departments</string>
    </property>
    <addaction name="actionCustomer_Relations"/>
    <addaction name="actionLab"/>
    <addaction name="actionFinishing"/>
    <addaction name="actionYield"/>
    <addaction name="actionProduct_Management"/>
    <addaction name="actionPackaging"/>
    <addaction name="actionDistillate"/>
    <addaction name="actionPOS"/>
    <addaction name="separator"/>
    <addaction name="actionPage_Switch_ON_OFF"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuIntake"/>
   <addaction name="menuPages"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="newBag">
   <property name="text">
    <string>Initial Intake</string>
   </property>
  </action>
  <action name="checkInventory">
   <property name="text">
    <string>Check Inventory</string>
   </property>
  </action>
  <action name="quit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionCustomer_Relations">
   <property name="text">
    <string>Customer Relations</string>
   </property>
  </action>
  <action name="actionLab">
   <property name="text">
    <string>Lab</string>
   </property>
  </action>
  <action name="actionFinishing">
   <property name="text">
    <string>Finishing</string>
   </property>
  </action>
  <action name="actionYield">
   <property name="text">
    <string>Yield</string>
   </property>
  </action>
  <action name="actionProduct_Management">
   <property name="text">
    <string>Product Management</string>
   </property>
  </action>
  <action name="actionPackaging">
   <property name="text">
    <string>Packaging</string>
   </property>
  </action>
  <action name="actionDistillate">
   <property name="text">
    <string>Distillate</string>
   </property>
  </action>
  <action name="actionPOS">
   <property name="text">
    <string>POS</string>
   </property>
  </action>
  <action name="actionPage_Switch_ON_OFF">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Page Switch ON/OFF</string>
   </property>
  </action>
  <action name="actionPost_Test">
   <property name="text">
    <string>Post Test</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
